experiments/tuning_examples/figures.ipynb

```diff
@@ -0,0 +1,362 @@
+{
```

Comment:
So far we've used to "experiments" directory only for replicating experiments from published papers. For this kind of thing, I'd prefer just documenting the gains with a simple test that both illustrates the feature and helps guarantee that we don't break it in the future.

Reply:
Oops. Removed.